pkgs/by-name/ca/cargo-shear/package.nix

```
@@ -0,0 +1,24 @@
+{ fetchFromGitHub, lib, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-shear";
+  version = "0.0.24";
+
+  src = fetchFromGitHub {
+    owner = "Boshen";
+    repo = pname;
```

```suggestion
    repo = "cargo-shear";
```

`repo = pname` is considered an anti-pattern.